### DISCUSSION TITLE
基于应用流转实现本机任意应用浮窗运行

### DISCUSSION THREAD

**Discussion by unbug** [eiyooooo/Easycontrol_For_Car#46] [Ideas]
我做了一个实验，在车机端填入车机的IP，竟然可以正常控制本机，这太棒了，就是我想要的实现车机**任意应用浮动窗口打开的效果**。
只不过遇到了比较大的问题：
1. 桌面显然无限循环了，这个无解，因为桌面就一个，所有用用都会在桌面上打开。
2. 那么如果是应用流转呢？很可惜，车机是 android 10，按照说明文档不支持无线场景下应用流转。

所以针对第 2 个问题，我没有找到很好的办法扫描出有线连接的本机，软件应该只支持 USB 端口的扫描。所以，问题是软件能否支持直接有线的方式添加本机呢？

当然，如果软件升级直接基于应用流转的思路实现所有应用浮窗运行就更好了，在车机上体验真的很棒，目前我在网上没有找到类似的解决方案，这个方案比分屏自由度高太多了。

**Comment by eiyooooo** [comment]
安卓无法实现有线连接本机，连接本机可在打开5555端口后可使用地址: 127.0.0.1:5555

安卓10不支持应用流转，若有**任意软件浮动打开**的需要可以试试 shizuku+[米窗](https://github.com/sunshine0523/Mi-Freeform/releases/tag/v2.5.18) 或者其它方案
